src/icmp/Icmp4.cc

```diff
@@ -146,10 +151,10 @@ Icmp4::SendEcho(Ip::Address &to, int opcode, const char *payload, int len)
 
     if (x < 0) {
         int xerrno = errno;
-        debugs(42, DBG_IMPORTANT, MYNAME << "ERROR: sending to ICMP packet to " << to << ": " << xstrerr(xerrno));
+        debugs(42, DBG_IMPORTANT, MYNAME << "ERROR: sending ICMP packet to " << to << ": " << xstrerr(xerrno));
```

Comment:
If we are going to fix this message, we should remove deprecated MYNAME prefix.

```suggestion
        debugs(42, DBG_IMPORTANT, "ERROR: sending ICMP packet to " << to << ": " << xstrerr(xerrno));
```